glib/Cargo.toml

```diff
@@ -36,6 +36,9 @@ thiserror = "1"
 gio_ffi = { package = "gio-sys", path = "../gio/sys", optional = true }
 memchr = "2.5.0"
 
+[target.'cfg(windows)'.dependencies]
+windows = { version = "0.44", features = ["Win32_Foundation"] }
```

Comment:
We've made the 'windows' crate optional for Gtk+ binding. If it's unconditionnal on glib, we should also change it in gtk.

otherwise, lgtm

Reply:
This is only used for the `GetLastError` call and I think this can be dropped considering `std::io::Error::last_os_error` may be used in its stead.

Reply:
Let's do that then?